cloudhub-2/modules/ROOT/pages/ps-config-domains.adoc

```diff
@@ -73,6 +73,17 @@ _Private DNS targets_ can only be reached from inside a private network, or thro
 
 When you add the TLS context, you can select the ciphers to use with the TLS context.
 
+[NOTE]
+====
+Starting Jan 15, 2025, CloudHub 2.0 private space ingress will no longer support TLS cipher version 1.1. Therefore, the following TLS 1.1 ciphers will be deprecated:
+```
+TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA
+TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA
+TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA
+TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA
+```
+====
+
```

Comment:
According to my understanding, the note should be written in simple present instead of future. See: https://confluence.internal.salesforce.com/display/DOCTEAM/forward-looking+statements%3A+Style+Guidelines